src/Daemon/BaseDaemon.cpp

```diff
@@ -418,7 +418,7 @@ class SignalListener : public Poco::Runnable
             SentryWriter::onFault(sig, error_message, stack_trace);
 
             /// Advice the user to send it manually.
-            if constexpr (std::string_view(VERSION_OFFICIAL).contains("official build"))
+            if (String(VERSION_OFFICIAL).contains("official build"))
```

Comment:
Extra allocation in the fatal signal handler is not good.

Reply:
There are a lot of allocations on strings formatting and logging, so I don't think it's critical. But I committed this change by mistake, it was to fix the "unreachable code" error when building locally